go-for-eat/screens/User.js

```diff
@@ -1,11 +1,23 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import { Text } from 'react-native';
+import PropTypes from 'prop-types';
 
 class User extends Component {
-
+  static propTypes = {
```

Comment:
Loving the fact that you are using propTypes, just if you are going to use them try to implement them in all the components :).